dashboard/modules/train/train_head.py

```diff
@@ -40,17 +48,35 @@ async def get_train_runs(self, req: Request) -> Response:
         stats_actor = await self.get_train_stats_actor()
 
         if stats_actor is None:
-            details = TrainRunsResponse(train_runs=[])
+            return Response(
+                status=500,
+                text=(
+                    "Not available. Please make sure Ray Train is running "
+                    "and that the Train state actor is enabled using the "
+                    "RAY_TRAIN_ENABLE_STATE_TRACKING environment variable."
+                ),
```

Comment:
nit: Clarify what is not available? And also explicitly call out that the value of `RAY_TRAIN_ENABLE_STATE_TRACKING` should be set to `1` to enable.